dc/s2n-quic-dc/src/uds/receiver.rs

```diff
@@ -0,0 +1,164 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use nix::{
+    fcntl::{fcntl, FcntlArg, OFlag},
+    sys::socket::{
+        bind, recvmsg, socket, AddressFamily, ControlMessageOwned, MsgFlags, SockFlag, SockType,
+        UnixAddr,
+    },
+    unistd::{unlink},
+};
+use std::{
+    os::{
+        fd::AsFd,
+        unix::io::{AsRawFd, RawFd},
+    },
+    path::{Path},
+};
+use tokio::io::{unix::AsyncFd, Interest};
+
+const BUFFER_SIZE: usize = u16::MAX as usize;
```

Comment:
I've used the same buffer size that the recv buffer in the TCP worker is using. ([link](https://github.com/aws/s2n-quic/blob/main/dc/s2n-quic-dc/src/stream/server/tokio/tcp/worker.rs#L50))